stdlib/3/curses/ascii.pyi

```diff
@@ -0,0 +1,69 @@
+from typing import Sequence, Union, overload
+
+NUL: int
+SOH: int
+STX: int
+ETX: int
+EOT: int
+ENQ: int
+ACK: int
+BEL: int
+BS: int
+TAB: int
+HT: int
+LF: int
+NL: int
+VT: int
+FF: int
+CR: int
+SO: int
+SI: int
+DLE: int
+DC1: int
+DC2: int
+DC3: int
+DC4: int
+NAK: int
+SYN: int
+ETB: int
+CAN: int
+EM: int
+SUB: int
+ESC: int
+FS: int
+GS: int
+RS: int
+US: int
+SP: int
+DEL: int
+
+controlnames: Sequence[int]
+
+def isalnum(c: str) -> bool: ...
+def isalpha(c: str) -> bool: ...
+def isascii(c: str) -> bool: ...
+def isblank(c: str) -> bool: ...
+def iscntrl(c: str) -> bool: ...
+def isdigit(c: str) -> bool: ...
+def isgraph(c: str) -> bool: ...
+def islower(c: str) -> bool: ...
+def isprint(c: str) -> bool: ...
+def ispunct(c: str) -> bool: ...
+def isspace(c: str) -> bool: ...
+def isupper(c: str) -> bool: ...
+def isxdigit(c: str) -> bool: ...
+def isctrl(c: str) -> bool: ...
+def ismeta(c: str) -> bool: ...
+@overload
```

Comment:
These could also be done with a TypeVar ranging over int and str. Using overloads is fine too though.